Samples/iOS-Swift/iOS-Swift-UITests/LaunchUITests.swift

```diff
@@ -92,6 +92,15 @@ class LaunchUITests: BaseUITest {
         XCTAssertTrue(0.5 > frozenFramesPercentage, "Too many frozen frames.")
     }
     
+    func testCorruptEnvelope() {
```

Comment:
pls add a comment with a link to GH-4280 and describing on a high level how the test works 🙏